pkg/apis/intevents/v1alpha1/brokercell_lifecycle_test.go

```diff
@@ -336,7 +337,7 @@ func TestMarkBrokerCellStatus(t *testing.T) {
 		wantType      apis.ConditionType
 		wantCondition corev1.ConditionStatus
 	}{{
-		name: "mark ingressReady unknown",
+		name: fmt.Sprintf("mark %s unknown", string(BrokerCellConditionIngress)),
```

Comment:
This makes sense, I agree that having a string to search when test fail is helpful for the debugging purpose (which I was not aware of before), and I guess this might be one of the reasons why we have names for each sub-UT. I'll change it.

Reply:
Ok. I see the value in having the searchable string. I have no objection to reverting the changes.